pyaedt/hfss.py

```diff
@@ -4435,22 +4499,22 @@ def validate_full_design(self, dname=None, outputdir=None, ports=None):
                 f.write("%s\n" % item)
         return val_list, validation_ok  # Return all the information in a list for later use.
 
-    @pyaedt_function_handler()
+    @pyaedt_function_handler(plot_name="plot", sweep_name="sweep", port_names="ports", port_excited="ports_excited")
     def create_scattering(
-        self, plot_name="S Parameter Plot Nominal", sweep_name=None, port_names=None, port_excited=None, variations=None
+        self, plot="S Parameter Plot Nominal", sweep=None, ports=None, ports_excited=None, variations=None
     ):
         """Create an S-parameter report.
 
         Parameters
         ----------
-        plot_name : str, optional
+        plot : str, optional
              Name of the plot. The default is ``"S Parameter Plot Nominal"``.
-        sweep_name : str, optional
+        sweep : str, optional
              Name of the sweep. The default is ``None``.
-        port_names : list, optional
+        ports : list, optional
              List of port names. The first index, i, in S[i,j].
              The default is ``None``. (include only self-terms)
```

Comment:
"(include only self-terms)" in unclear. This should be clarified as a complete and made a complete sentence.